.github/DISCUSSION_TEMPLATE/request-help.yml

```diff
@@ -1,4 +1,13 @@
 body:
+  - type: dropdown
+    id: question-type
+    attributes:
+      label: What would you like help with?
```

Comment:
Both the category ("**Request** help") as well as this "would like help with" phrasing are intentionally trying to set a good tone

Reply:
Sounds good to me! 🙃